.github/workflows/eval.yml

```
@@ -98,10 +98,15 @@ jobs:
                 run_id,
                 name: `merged-${system}`
               })
-              if (result.data.total_count > 0) return
+              if (result.data.total_count > 0) {
+                core.setOutput('targetRunId', run_id)
+                return
+              }
               await new Promise(resolve => setTimeout(resolve, 5000))
             }
-            throw new Error(`No merged-${system} artifact found.`)
+            // No artifat found at this stage. This means that Eval failed on the target branch.
+            // This can only happen when Eval is broken on the target branch and this PR fixes it.
```

Are your confident this is the __only__ possible scenario, or should we say "usually"?

Also minor typo `artifat` → `artifact`.

```suggestion
            // No artifact found at this stage. This usually means that Eval failed on the target branch.
            // This should only happen when Eval is broken on the target branch and this PR fixes it.
```

> and this PR fixes it.

Could it also happen if there's an unrelated eval failure on the target branch, (so the PR doesn't fix it)?